src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -25,6 +25,7 @@ const DomainList = (props) => {
     DefaultImage,
     isCommunity,
     isBlockedSection,
+    isDomain,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isDomain` is introduced but there's no validation or type checking for this prop. It would be good to add PropTypes for this component to ensure the correct types are being passed in.
```diff
+  DiscoveryItemList.propTypes = {
+    ...
+    isDomain: PropTypes.bool,
+    ...
+  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->